app/scripts/Chromosome2DAnnotations.js

```diff
@@ -4,11 +4,20 @@ import * as PIXI from 'pixi.js';
 import PixiTrack from './PixiTrack';
 import ChromosomeInfo from './ChromosomeInfo';
 
+import pubSub from './services/pub-sub';
+
+// Maximum delay in ms between mousedown and mouseup that is registered as a
+// click. Note we need to use mousedown and mouseup as PIXI doesn't recognize
+// click events with out current setup. Since most UIs treat long clicks as
+// either something special or a cancelation we follow best practices and
+// implement a threshold on the delay as well.
+const MAX_CLICK_DELAY = 300;
```

Comment:
> Since most UIs treat long clicks as either something special or a cancelation we follow best practices and implement a threshold on the delay as well.

In general you're right but then there's no way to cancel a click. Feel free to propose an alternative.

> Making the whole thing inherit from some base class might be hard, but could this value be stored in one place?

I'll move `MAX_CLICK_DELAY` to `config/index.js`